invenio_app_ils/config.py

```diff
@@ -1180,20 +1180,10 @@ def _(x):
             state=dict(terms=dict(field="state")),
             delivery=dict(terms=dict(field="delivery.method")),
             returns=overdue_agg,
-            availability=dict(
```

Comment:
@topless @ntarocco  this was one of the requirements, it made a lot of sense to have this filter, so I think we should discuss about alternative solutions

Reply:
ok, we can brainstorm this because I don't have ideas at the moment

Reply:
@kprzerwa I tried to come up with an alternative so we can keep the feature, but I failed. When I was reviewing the changes, this `availability` filter, I found it hard to understand what was the purpose of it. But since its a requirement we have to work around it.

Reply:
yes I know it is tricky. I created an issue for this to discuss and tackle later.